tests/peer/test_peermetadata.py

```diff
@@ -5,7 +5,7 @@
 
 # Testing methods from IPeerMetadata base class.
 
-
+# pylint: disable=duplicate-code
```

Comment:
Is this still needed after the bump up in min lines of similarity?

Reply:
removed